dat/ships/mule.xml

```diff
@@ -14,7 +14,7 @@
  <trail_generator x="-33" y="0" h="0">fire</trail_generator>
  <fabricator>Melendez Corp.</fabricator>
  <license>Large Civilian Vessel</license>
- <description>A heavy freighter specialized in transporting cargo throughout the galaxy. Also has sufficient accommodations for passengers depending on the model, allowing for enjoyable cruises. The Mule is a favourite target of pirates, so avoiding dangerous systems is strongly advised.</description>
+ <description>A heavy freighter specialised in transporting cargo throughout the galaxy. Also has sufficient accommodation for passengers, depending on the model, allowing for enjoyable cruises. The Mule is a favourite target of pirates, so avoiding dangerous systems is strongly advised.</description>
```

Comment:
Although we do use British English, I think we pretty much stick to [Oxford Spelling](https://en.wikipedia.org/wiki/Oxford_spelling) which means using endings in `-ize` instead of `-ise`.

Reply:
I'll remember to check for it that way in future... I have a niggling that I might have seen one or two in the ship descriptions... *sigh* No time now, maybe later! :)
I had noticed the use of British English spelling but I'm a little dubious about the punctuation... I only "corrected" it if there were other changes I definitely wanted to make!